### PR TITLE
Feat/clubsnav

### LINKED ISSUE #6: Subnavigation
As seen on https://www.888.de/888club/Vorteile/
Currently there are 4 block-style links under the header.

### PR DESCRIPTION
**Pull request by kailasnadh790**
## Ticket/Issue number(s)
Fix #6 

## Test URLs
- Before: https://main--888de--aemsites.hlx.page/888club/
- After: https://feat-clubsnav--888de--aemsites.hlx.page/888club/

### CODE REVIEW

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ function createMobileMenu(block) {
     details.className = 'accordion-item';
     const summary = document.createElement('summary');
     summary.className = 'accordion-header';
-    summary.innerHTML = accordionItem.querySelector('strong').outerHTML;
+    summary.innerHTML = accordionItem.querySelector('strong') ? accordionItem.querySelector('strong').outerHTML : '';
```
Comment (helms-charity) [inline blocks/footer/footer.js:13]: I think this is causing an anomaly on the struktur-level page. Sometimes the accordion is broken when I navigate there, and always if I refresh it's broken. The other club pages don't do that.
image
https://feat-clubsnav--888de--aemsites.hlx.page/888club/level-struktur

Comment (kailasnadh790) [reply blocks/footer/footer.js]: @helms-charity refactored the code...looks like that resolved the issue.

**Review by dave-fink — APPROVED** [review]
LGTM

**Review by helms-charity — APPROVED** [review]
Looks good now

**Pull request merged**